commands/MakeModel.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * @adonisjs/lucid
+ *
+ * (c) Harminder Virk <virk@adonisjs.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+*/
+
+import { join } from 'path'
+import { BaseCommand, args } from '@adonisjs/ace'
+
+export default class MakeModel extends BaseCommand {
+  public static commandName = 'make:model'
+  public static description = 'Make a new Lucid model'
+
+  /**
+   * The name of the model file.
+   */
+  @args.string({ description: 'Name of the model class' })
+  public name: string
+
+  /**
+   * Pull path from the `models` directory declaration from
+   * the `.adonisrc.json` file or fallback to `app/Models`
+   */
+  protected $getDestinationPath (): string {
+    const rcContents = this.application.rcFile
+
+    return rcContents.namespaces['models'] ?? 'app/Models'
```

Comment:
Here you are using the `namespace` as the directory. However, you will have to resolve the directory for the given namespace. Assembler already does this https://github.com/adonisjs/assembler/blob/develop/commands/Make/Base.ts#L41